FinniversKit/Sources/Base Components/ButtonStyle.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+public struct InlineFlatStyle: ButtonStyle {
```

Comment:
Should we perhaps move button styles to FinnUI? I know it's not the right place for it, but at least we can keep our SwiftUI views and other components in one place. In general, I feel like it's time to introduce the new framework.

Reply:
Mmm, I am not sure, we could.

If so, then we should not have swiftUI code within FinniversKit at all. Maybe just depend on it to get the assets/fonts, meaning that Color and Font overrides should be part of FinnUI too.

In that sense, I would keep the stuff in FinniversKit in this PR and move them in a new one, when we have a clear
plan for it

Reply:
What this new SwiftUI framework needs from FinniversKit is just he DNA part like fonts, colours and assets. It's doesn't really need UIKit components like Button and Label. That would probably resolve some of the conflicts like Finnivers/SwiftUI Button type. 

But yes, I agree. Let's discuss this further and prepare a clear plan for it.